flycheck.el

```diff
@@ -6068,6 +6069,33 @@ See URL `http://www.foodcritic.io'."
        ;; http://matschaffer.github.io/knife-solo/#label-Init+command
        (locate-dominating-file parent-dir "cookbooks")))))
 
+(flycheck-define-checker chicken-scheme
```

Comment:
I think the syntax checker should probably rather be named "scheme-chicken", given that the language it actually compiles is Scheme, and chicken is just the name of a Scheme compiler.